lib/activate.js

```diff
@@ -50,6 +50,32 @@ async function start(email, password, alf, verification, emailPassword) {
   ]);
 
   try {
+    console.log('[INFO] Check Follow...');
+
+    await page.waitForNavigation();
+
+    const needFollowJump = await page.$('.g6.connect-scan-group');
+
+    if (needFollowJump) {
+      let loginBtn = await page.$('a[rel="nofollow"]');
+      loginBtn.click();
+
+      await page.waitForNavigation({ waitUntil: 'load' });
+
+      await page.waitForSelector(".g12.phone-login-box.clear.p20");
+
+      //can't click by DOM API
+      let emailLogin = await page.$('a[data-event="toMailLogin"]');
+
+      let aBox = await emailLogin.boundingBox();
+
+      await page.mouse.move(aBox.x + 100, aBox.y + 25);
+      await page.mouse.down();
+      await page.mouse.up();
+    } else {
+      console.log("[INFO] Skip 'Check Follow'...");
```

Comment:
I prefer to print a log in the `if` clause, and get rid of the `else` clause. That way we are able to debug everything step by step. For example:

```js
await page.waitForNavigation();

const needFollowJump = await page.$('.g6.connect-scan-group');

// --- Assumed above operations cause error

if (needFollowJump) {
  // With the log, we can identify the error immediately!
  //console.log("[INFO] Skipping 'Check Follow'...");

  // Or below operations cause error, we have no idea.
  let loginBtn = await page.$('a[rel="nofollow"]');
}
```

WDYT?

Reply:
😉